templates/e3sm_diags.bash

```diff
@@ -83,9 +120,35 @@ param.output_format_subplot = {{ output_format_subplot }}
 param.multiprocessing = {{ multiprocessing }}
 param.num_workers = {{ num_workers }}
 
+# Set specific parameters for new sets
+enso_param = EnsoDiagsParameter()
+enso_param.reference_data_path = '{{ obs_ts }}'
+enso_param.test_data_path = test_ts
+enso_param.test_name = short_name
+enso_param.test_start_yr = start_yr
+enso_param.test_end_yr = end_yr
+enso_param.ref_start_yr = "1990"
+enso_param.ref_end_yr = "1999"
```

Comment:
ENSO diags work fine with these years. Should I change it to use the same years as QBO though?